### PR TITLE
Feature/fix bin dev

### PR DESCRIPTION
**Pull request by cchaudier**
## Bug

bin/dev fail after lint correction

## Fix

In bin/dev we build a command line.
So we must ignore the shellsheck SC2086 rule to split on spaces
The docker image dind used in gitlab-ci don't contain bash.              
We add bash because it used on shell scripts launch in the CI jobs.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ stages:
 
 # Before executing a job's script, we check the output of docker info
 before_script:
+  - apk add --update bash
```
Comment (jmaupetit) [inline .gitlab-ci.yml]: Why do we need to update bash?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -30,6 +31,7 @@ build:image:
   script:
     - bin/build
     - mkdir -p docker/images
+    - ARNOLD_IMAGE="arnold:$(tr -d '\n' < VERSION)"
```
Comment (jmaupetit) [inline .gitlab-ci.yml]: Why not using YAML anchors to avoid repetition?

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -30,6 +31,7 @@ build:image:
...
     - docker save -o docker/images/arnold.tar ${ARNOLD_IMAGE}
   # Store the built as an artifact that can be reused in other jobs
   artifacts:
```
Comment (jmaupetit) [inline .gitlab-ci.yml]: BTW, I think these changes should not be part of this PR (out of scope).

File: bin/dev [hunk 4]
```diff
@@ -50,7 +50,8 @@ function _run-minishift() {
     fi
 
     echo "Starting minishift with options: ${opts}"
-    minishift start "${opts}"
+    # shellcheck disable=SC2086
+    minishift start ${opts}
```
Comment (jmaupetit) [inline bin/dev:54]: :+1:

File: .gitlab-ci.yml (hunk 1 above)
Comment (cchaudier) [reply .gitlab-ci.yml]: It's explain on the PR message, it's for use the build script.

File: .gitlab-ci.yml (hunk 2 above)
Comment (cchaudier) [reply .gitlab-ci.yml]: No time for this for the moment but yes we must use anchors.
I'm open an issue for sync circleCI and gitlab-ci files.

File: .gitlab-ci.yml (hunk 3 above)
Comment (cchaudier) [reply .gitlab-ci.yml]: You're right.
I'm open an issue for sync circleCI and gitlab-ci files.

File: .gitlab-ci.yml (hunk 1 above)
Comment (jmaupetit) [reply .gitlab-ci.yml]: Can you add a comment instead?

Comment (cchaudier) [reply .gitlab-ci.yml]: I'm going to do this on the issue #59.
I've remove the commit for gitlab-ci.

**Review by jmaupetit — APPROVED** [review]
LGTM!
